modules/installation-vsphere-infrastructure.adoc

```diff
@@ -24,3 +24,9 @@ clusters are subject to the following restrictions:
 
 If you use a vSphere version 6.5 instance, consider upgrading to 6.7U2 before
 you install {product-title}.
+
+[IMPORTANT]
+====
+Time configuration of ESXi hosts is critical for a successful installation of
+OpenShift Container Platform.  See link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-8756D419-A878-4AE0-9183-C6D5A91A8FB1.html[Edit Time Configuration for a Host] in the VMware documentation.
```

Comment:
```suggestion
{product-title}. See link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-8756D419-A878-4AE0-9183-C6D5A91A8FB1.html[Edit Time Configuration for a Host] in the VMware documentation.
```

Reply:
@kalexand-rh 
Sorry I didn't see the comments until now.  I will fix the {product-title}
It should be all supported versions.